src/theory/strings/inference_manager.h

```diff
@@ -210,18 +210,23 @@ class InferenceManager
    * exists, otherwise it returns null.
    */
   Node getProxyVariableFor(Node n) const;
-  /** register length
+  /** register term
+   *
+   * This method is called on non-constant string terms n. It returns a lemma
+   * that ensures that the length n satisfies its assigned status (given by
+   * argument s), or null if no lemma is necessary.
```

Comment:
This is not an issue introduced by this PR but I just realized that the comment here is not accurate: there is no argument `s`.